package.json

```diff
@@ -29,16 +29,16 @@
     "start": "yarn build:dev",
```

Comment:
Not sure it was meant to be in this PR, it's mixing upgrades to packages and new packages needed for typescript. It may be correct though

Reply:
Actually, there are no changes on this line. Or maybe I miss something...

Reply:
There are some changes but nothing blocking :)